src/test.js

```diff
@@ -78,7 +79,8 @@ Test.prototype = {
 		runLoggingCallbacks( "testStart", {
 			name: this.testName,
 			module: this.module.name,
-			testId: this.testId
+			testId: this.testId,
+			bad: this.wasBad
```

Comment:
`wasBad` is okay, but `bad` is a poor variable name. Might as well use `wasBad` for the logging callback.